src/kernels/jupyter/jupyterUtils.ts

```diff
@@ -133,8 +138,18 @@ export function createRemoteConnectionInfo(
     };
 }
 
-export function computeServerId(uri: string) {
-    return hashjs.sha256().update(uri).digest('hex');
+export async function computeServerId(uri: string) {
+    try {
+        const inputBuffer = new TextEncoder().encode(uri);
+        const hashBuffer = await crypto.subtle.digest({ name: 'SHA-256' }, inputBuffer);
```

Comment:
The difference is the digest function is a promise. That rippled all over the place.